packages/test/package.json

```diff
@@ -25,18 +25,36 @@
   "author": "Temporal Technologies Inc. <sdk@temporal.io>",
   "license": "MIT",
   "dependencies": {
+    "@grpc/grpc-js": "^1.6.8",
```

Comment:
Did we have to move all of the shared dev dependencies from the root package?

Reply:
Yes, when I remove, I get: 
<img width="1060" alt="image" src="https://user-images.githubusercontent.com/251288/194670287-f67bff2c-8f82-4933-87fb-1aa034f186bd.png">

Reply:
That means several dependencies can now be pruned from the top level package.json... `depcheck` might help with that.

Reply:
This can be done later, but we will definitely have to find an efficient way of ensuring version coherence between dependencies of the various subprojects. There are tools for this, but haven't used any yet.